cmd/server/assets/realmadmin/_stats_keyserver.html

```diff
@@ -100,10 +100,26 @@ <h5 class="modal-title">Key server stats</h5>
     .done(function(data, status, xhr) {
       $slider.attr('min', Math.min(smoothing, data.length));
       $slider.attr('max', data.length);
+      $slider.attr('value',data.length);
+      $slider.attr('step', 1);
       $slider.val(data.length);
       $onsetSlider.attr('min', Math.min(smoothing, data.length));
       $onsetSlider.attr('max', data.length);
+      $onsetSlider.attr('value',data.length);
+      $onsetSlider.attr('step', 1);
       $onsetSlider.val(data.length);
+  
+      $("#tektickmarks").empty();
+      for (i = Math.min(smoothing, data.length); i < data.length; i++) {
+        let opt = '<option value="' + i + '" label="' + data[i].day.substring(0,10) + '"></option>';
+        $("#tektickmarks").append(opt);
+      }
+
+      $("#onset_slide_ticks").empty();
```

Comment:
every time you do `$('#onset_slide_ticks')` jquery is retrieving it from the DOM. I'd just make this once and reference a variable here.